### PR TITLE
Improved Dockerfiles.

### PR DESCRIPTION
**Pull request by partikus**
Switched Phing installation from pear to latest-build download.
Resolved issue with file permissions during entrypoint.sh build.
Enabled output of phing list task.

### CODE REVIEW

**Comment by partikus** [comment]
@jhryniuk CR please

**Comment by partikus** [comment]
@dominikpgs @kubawerlos @tarnawski CR please

File: build.xml [hunk 1]
```diff
@@ -1,16 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project name="demo" default="build">
-    <target name="build"
-            depends="list"
-    />
-
+<project name="demo" default="list">
     <target name="list"
-            unless="list.done"
```
Comment (jhryniuk) [inline build.xml]: Why you getting rid of this condition?

File: build.xml [hunk 2]
```diff
@@ -1,16 +1,8 @@
...
             description="List file structure">
-        <exec executable="ls">
-            <arg value="-l"/>
-            <arg value="-a"/>
-        </exec>
+        <exec command="ls -la" logoutput="true"/>
 
         <property name="list.done" value="true"/>
```
Comment (jhryniuk) [inline build.xml]: this property is useless now.

**Review by jhryniuk — APPROVED** [review]

**Pull request merged**